apps/maestro/src/ui/pages/InterchainTokenDetailsPage/TokenDetailsSection.tsx

```diff
@@ -82,7 +158,51 @@ const TokenDetailsSection: FC<TokenDetailsSectionProps> = (props) => {
         {maskAddress(tokenAddress)}
       </CopyToClipboardButton>,
     ],
-    ...(wallet && !isSuiChain && !isStellarChain
+    ...(connectedAddress && wallet && isHederaChain
+      ? [
+          [
```

Comment:
It'd also be a nice opportunity to refactor this... The whole ternary with spreading an empty array is counter-intuitive.
What do you think of just initializing the array with what's necessary, and then push stuff into it inside if conditions?